dataanalysis/core.py

```diff
@@ -15,7 +15,12 @@
 import time
 import glob
 import socket
-from collections import Mapping, Set, Sequence, OrderedDict
+try:
+    from collections.abc import Mapping, Set, Sequence, Iterable
+    from collections import OrderedDict
```

Comment:
I changed the import call and added the import of OrderedDict, which is still imported as usual in the new version.